spec/cassettes/DqtApi/_find_trn_/1_1_1.yml

```diff
@@ -0,0 +1,203 @@
+---
+http_interactions:
+  - request:
+      method: get
+      uri: https://qualified-teachers-api-preprod.london.cloudapps.digital/v2/teachers/find?dateOfBirth&emailAddress=kevin@kevin.com&firstName=kevin&ittProviderName=provider&lastName&nationalInsuranceNumber=1000000
+      body:
+        encoding: US-ASCII
+        string: ''
+      headers:
+        User-Agent:
+          - Faraday v1.10.0
+        Authorization:
+          - Bearer kTWC7StTAeEKBpMz
```

Comment:
Shouldn't this be `Bearer testKey`? Were the tapes generated before we changed the `.env` files?